### ISSUE TITLE
Problems when using vim-tmux-navigator with fig (https://fig.io)

### ISSUE THREAD

**Issue by JULO01** [christoomey/vim-tmux-navigator#339]
### Vim-Tmux Problems when using fig
As mentioned [here](https://github.com/christoomey/vim-tmux-navigator/issues/317#issuecomment-1310419779), there are problems when using vim-tmux-navigator together with [fig](https://fig.io).

**My observations:** 
- **Switching panels inside tmux itself works great** (even if there is nvim running in one of that panes)
- Trying to switch windows **inside nvim** (with keybindings) **only triggers switching the tmux panes**
- Manually calling :TmuxNavigateRight(/Left/Up/Down) in nvim command mode **works as expected** (for nvim windows, tmux panes, and between those, see gif below)
- Therefore the main problem (as far as I interpreted this right): **Tmux captures the keybindings when they are pressed inside nvim, but fails to send them back to nvim** 

![capture](https://user-images.githubusercontent.com/90795588/212475015-7e528372-480e-4b52-b4ab-a99c25e7cf72.gif)

**Possible problem sources:**
```bash
#.zshrc
# Fig pre block. Keep at the top of this file.
[[ -f "$HOME/.fig/shell/zshrc.pre.zsh" ]] $$ builtin source "$HOME/.fig/shell/zshrc.pre.zsh"

# My zsh configs
# ...

# Fig post block. Keep at the bottom of this file.
[[ -f "$HOME/.fig/shell/zshrc.post.zsh" ]] $$ builtin source "$HOME/.fig/shell/zshrc.post.zsh"
```

```bash
# ~/.fig/shell/zshrc.pre.zsh
# (sourced in the .zshrc)
[ -x ~/.local/bin/fig ] && eval "$(~/.local/bin/fig init zsh pre --rcfile zshrc)"
```

```bash
# ~/.fig/shell/zshrc.post.zsh
# (sourced in the .zshrc)
[ -x ~/.local/bin/fig ] && eval "$(~/.local/bin/fig init zsh post --rcfile zshrc)"
```

**The setup:**
- `tmux -V` --> tmux 3.3a
- `nvim -version` --> NVIM v0.8.1
- `:TmuxNavigatorProcessList (inside nvim)` -->  Ss+ zsh (figterm) 
- `sw_vers` --> macOS 12.6

If anyone has a solution in mind, please tell me about it :) 
If not, I'll just say goodbye to fig, that's fine to me...

And if more information is needed, just hit me up and I'll paste it here.

**Comment by vhristev** [comment]
After uninstalling `Fig` everything started to work again. 

```
brew uninstall fig
```

**Comment by JULO01** [comment]
> After uninstalling `Fig` everything started to work again.
> 
> ```
> brew uninstall fig
> ```

@vhristev Yes, that solved the problem for me too. The question is, if there is any solution/ workaround to have fig and the tmux-navigator working at the same time, without having to dispense with one of both..

**Comment by JakeTompkins** [comment]
I'm having the same problem. I'd like to keep using fig so a solution to this would be awesome. Let me know if I can provide any other diagnostics to help troubleshoot

**Comment by nguyenanhhao221** [comment]
I'm having the same issue. Removing fig work fine. 

But does anybody know a similar solution to fig. I'm not that familiar with all the terminal command so fig tool can really help with it cause they also include a explanation

**Comment by unitythemaker** [comment]
I have the same problem but I don't have Fig installed. I use NixOS if that makes any change but I am so annoyed by how it works perfect on everyone else and doesn't work on my pc...

**Comment by hoolek77** [comment]
If you still encounter this issue, this solution helped me to fix integration with Fig https://github.com/christoomey/vim-tmux-navigator/issues/295#issuecomment-922092511

**Comment by JULO01** [comment]
As @hoolek77 mentioned, the solution of [#295 ](https://github.com/christoomey/vim-tmux-navigator/issues/295#issuecomment-922092511) fixes this issue (mostly). After creating the `is_vim.sh` as suggested, and replacing the is_vim line in the tmux plugin, the horizontal pane switching does work, although vertical window switching in vim itself does still not work as expected. 
I will **close this issue** as fixed with the workaround of [#295 ](https://github.com/christoomey/vim-tmux-navigator/issues/295#issuecomment-922092511).
